### PR TITLE
docs: add contributing spanish section

### PR DESCRIPTION
**Pull request by VicenteMerino**
Translation of contributing section for spanish language

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 639b3d9db4aa0f5a96992c662712e064e54b5edf at: https://64b0b913d03a707382b3daf6--fastapi.netlify.app

**Review by jfunez — APPROVED** [review]

File: docs/es/docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+#### Consejos y guías
+
+* Revisa las <a href="https://github.com/tiangolo/fastapi/pulls" class="external-link" target="_blank">pull requests existentes</a> para tu idioma y agrega revisiones solicitando cambios o aprobándolos.
+
+!!! tip "Consejo"
+    Puedes <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request" class="external-link" target="_blank">agregar comentarios con sugerencias de cambios</a> a las pull requests existentes.
+
+    Revisa la documentación sobre <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews" class="external-link" target="_blank">cómo agregar una revisión de pull request</a> para aprobarla o solicitar cambios.
+
+* Revisa las < href="https://github.com/tiangolo/fastapi/discussions/categories/translations" class="external-link" target="_blank">GitHub Discussions</a> para coordinar las traducciones para tu idioma. Puedes suscribirte a ella, y cuando haya una nueva pull request para revisar, se agregará un comentario automático a la discusión.
```
Comment (setejera) [inline docs/es/docs/contributing.md:265]: Hola! En el a tag falta la a en el de apertura

**Review by setejera — CHANGES_REQUESTED** [review]
Pequeño error con un elemento a tag, linea 265

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**